### PR TITLE
Fix memcache configuration in Swift to include all nodes in cluster

### PR DESCRIPTION
**Pull request by ryshah**

### CODE REVIEW

File: roles/swift-proxy/templates/etc/swift/proxy-server.conf [hunk 1]
```diff
@@ -83,9 +83,6 @@ use = egg:swift#container_sync
 [filter:dlo]
 use = egg:swift#dlo
 
-[filter:staticweb]
-use = egg:swift#staticweb
-
```
Comment (ryshah) [inline roles/swift-proxy/templates/etc/swift/proxy-server.conf:88]: This is duplicate entry in the configuration file

**Comment by msambol** [comment]
LGTM. Good catch finding the duplicate `staticweb`.

**Comment by MaheshIBM** [comment]
I have reverted this merge.

**Comment by MaheshIBM** [comment]
I thought the pull request was for keystone-ldap but looks like its for master branch, so reverted it.

**Comment by MaheshIBM** [comment]
Ryshah, can you resubmit the pull request? I did not know i had access to merge and merged by mistake to master, sorry for the trouble.

**Pull request merged**